src/components/footer/footer.tsx

```diff
@@ -0,0 +1,15 @@
+import {ReactNode} from 'react';
+
+type ChildrenType = {
```

Comment:
Паттерн для пропсов это имя компонента + Props либо просто Props
```suggestion
type FooterProps = {
```